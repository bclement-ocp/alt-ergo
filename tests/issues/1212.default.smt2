; The generated model is incorrect. It should ensure that a and b are
; equal everywhere except on 0. This bug should be resolved
; after addressing https://github.com/OCamlPro/alt-ergo/issues/929

(set-option :produce-models true)
(set-logic ALL)
(declare-const a (Array Int Int))
(declare-const b (Array Int Int))
(assert (= (store a 0 0) (store b 0 0)))
(check-sat)
(get-model)
