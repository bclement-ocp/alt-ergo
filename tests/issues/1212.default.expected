
unknown
(
  (define-fun a () (Array Int Int) (as @a3 (Array Int Int)))
  (define-fun b () (Array Int Int) (as @a2 (Array Int Int)))
)
